evalml/automl/automl_search.py

```diff
@@ -764,6 +757,18 @@ def _post_evaluation_callback(self, pipeline, evaluation_results):
             self.add_result_callback(self._results['pipeline_results'][pipeline_id], pipeline, self)
         return pipeline_id
 
+    def _check_for_high_variance(self, pipeline, cv_scores, threshold=0.2):
+        """Checks cross-validation scores and logs a warning if variance is higher than specified threshhold."""
+        pipeline_name = pipeline.name
+        high_variance_cv = False
+
+        if cv_scores.mean() != 0:
```

Comment:
What case is this capturing?

Reply:
Hm, this was more a rework of what was already there and the fear of `cv_scores.mean() == 0` meaning you divide by 0 in the lines that follow... but it seems like numpy just handles division by 0 as nan, and nan won't ever be greater than the threshold so maybe we don't need this after all. I'll remove!